protocols/AnnotateVcf.sh

```diff
@@ -90,7 +90,12 @@ else
 		for i in {1..22}
 		do
 			#shellcheck disable=SC2129
-			echo -e "\n[[annotation]]\nfile=\"${gnomADGenomesAnnotation}/gnomad.genomes.r2.0.2.sites.chr${i}.normalized.vcf.gz\"\nfields=[\"AF_POPMAX\",\"segdup\"]\nnames=[\"gnomAD_genome_AF_MAX\",\"gnomAD_genome_RF_Filter\"]\nops=[\"self\",\"self\"]" >> "${vcfAnnoGnomadGenomesConf}"
+			if [[ $(zcat "${gnomADGenomesAnnotation}/gnomad.genomes.r2.0.2.sites.chr${i}.normalized.vcf.gz" | wc -l) == 0 ]]
```

Comment:
Weet niet hoe groot dit kan worden, maar wellicht is het handig om er head tussen te zetten. Als je 10 regels gezien hebt weet je genoeg en hoef je de rest van die VCF niet meer uit te pakken om regels te tellen. 
```suggestion
			if [[ $(zcat "${gnomADGenomesAnnotation}/gnomad.genomes.r2.0.2.sites.chr${i}.normalized.vcf.gz" | head | wc -l) == 0 ]]
```